contrib/babelfishpg_tsql/src/pltsql_utils.c

```diff
@@ -419,7 +419,7 @@ pltsql_createFunction(ParseState *pstate, PlannedStmt *pstmt, const char *queryS
  * And length should be restricted to 4000 for sys.varchar and sys.char datatypes
  */
 void
-pltsql_check_or_set_default_typmod(TypeName *typeName, int32 *typmod, bool is_cast)
+pltsql_check_or_set_default_typmod(TypeName *typeName, int32 *typmod, bool is_cast, bool is_procedure_or_func)
```

Comment:
Please update above comment to reflect the extra arg we are adding.

Reply:
Done